File: audiobook/src/main/java/de/ph1b/audiobook/interfaces/MultiPaneInformer.java
package de.ph1b.audiobook.interfaces;

/**
 * A basic interface that informs if the current layout mode has multiple panels.
 *
 * @author Paul Woitaschek
 */
public interface MultiPaneInformer {

    /**
     * @return true if the layout has multiple panels.
     */
    boolean isMultiPanel();
}

File: audiobook/src/main/kotlin/de/ph1b/audiobook/interfaces/MultiPaneInformer.kt
package de.ph1b.audiobook.interfaces

/**
 * A basic interface that informs if the current layout mode has multiple panels.

 * @author Paul Woitaschek
 */
interface MultiPaneInformer {

    /**
     * true if the layout has multiple panels.
     */
    fun isMultiPanel(): Boolean
}
